### PR TITLE
docs: add Leonardo Zizzamia to GDE resources

### PR DESCRIPTION
**Pull request by Zizzamia**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
This adds Leonardo Zizzamia to the list of Angular GDEs on the Angular Docs page -> cc @StephenFluin

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -663,5 +663,14 @@
...
     "group": "GDE"
+  },
+
+  "leonardozizzamia": {
+    "name": "Leonardo Zizzamia",
+    "picture": "leonardo.jpg",
+    "twitter": "zizzamia",
+    "website": "https://medium.com/@zizzamia",
+    "bio": "Leonardo is a Co-Founder of Plan, the command center for work life. He is deeply passionate about web performance and most recently developed Perfume.js to help companies prioritize roadmaps and business, through performance analytics. From 2018, co-organizer of the Angular San Francisco Meetup group.",
+    "group": "Angular"
```
Comment (StephenFluin) [inline aio/content/marketing/contributors.json]: This should be 'GDE'

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (Zizzamia) [reply aio/content/marketing/contributors.json]: Thank you @StephenFluin

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Can you squash these two commits?

https://twitter.com/stephenfluin/status/1009904095073718275

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Update git commit message to `docs: add Leonardo Zizzamia to GDE resources`

**Comment by brandonroberts** [comment]
@Zizzamia will you rebase on master and update the git commit message?

**Review by StephenFluin — APPROVED** [review]

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
@Zizzamia will you rebase again to fix the merge conflict? Thanks.

**Pull request closed without merging**